shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/model/TokenApiModel.kt

```diff
@@ -0,0 +1,28 @@
+package co.nimblehq.blisskmmic.domain.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class TokenApiModel(
+    @SerialName("access_token")
+    val accessToken: String,
+    @SerialName("token_type")
+    val tokenType: String,
+    @SerialName("expires_in")
+    val expiresIn: Int,
+    @SerialName("refresh_token")
+    val refreshToken: String,
+    @SerialName("created_at")
+    val createdAt: Int
+)
+
+fun TokenApiModel.toToken(): Token {
+    return Token(
+        accessToken,
+        tokenType,
+        expiresIn,
+        refreshToken,
+        createdAt
+    )
+}
```

Comment:
Optional: This could be rewritten to simply it a bit

```suggestion
fun TokenApiModel.toToken() = Token(
    accessToken,
    tokenType,
    expiresIn,
    refreshToken,
    createdAt
)
```